config/locales/en.yml

```diff
@@ -64,7 +64,7 @@ en:
     error:
       generic: |
         Oops! Something went wrong and we couldn't process that action. You
-        might want to try it again, or if that doesn't work, refresh the page.
+        might want to try it again.
```

Comment:
I see your point – there's no "app chrome", so no obvious place to put a new control within the app. Maybe it is best to simply remove the "refresh" wording, then. I'll leave the decision to you folks. Like I said, maybe I'm worrying too much about something that will never happen.